seqr/views/apis/data_manager_api.py

```diff
@@ -395,6 +396,69 @@ def load_rna_seq_sample_data(request, sample_guid):
     return create_json_response({'success': True})
 
 
+def _load_phenotype_prioritization(file_path, user):
+    data_by_id = load_phenotype_prioritization_data_file(file_path)
+
+    all_samples = [sample for project_samples in data_by_id.values() for sample in project_samples.values()]
+    all_records = [rec for sample_records in all_samples for rec in sample_records]
+    message = f'Parsed {len(all_records)} LIRICAL/Exomiser data records in {len(all_samples)} samples'
```

Comment:
computing out the totals is hard to read and doesn't add much value. I think it would be better to log here how many projects were parsed, and then have a separate log command for each project with the counts